src/compiler/checker.ts

```diff
@@ -31272,6 +31272,7 @@ namespace ts {
         function checkAwaitExpression(node: AwaitExpression): Type {
             // Grammar checking
             if (produceDiagnostics) {
+                const containingFunctionOrClassStaticBlock = getContainingFunctionOrClassStaticBlock(node);

```

Comment:
Can probably use a shorter name to make the `if` on line 31301 easier to read.
```suggestion
                const container = getContainingFunctionOrClassStaticBlock(node);
```